app/services/food_service.rb

```diff
@@ -0,0 +1,10 @@
+class FoodService
+
+  def self.food_search(search_terms)
+    food_api = ENV["food_api"]
+    response = Faraday.get("https://api.nal.usda.gov/fdc/v1/foods/search?api_key=#{food_api}&query=#{search_terms}")
+    body = response.body
+    JSON.parse(body, symbolize_names: true)
+  end
```

Comment:
You've got all the parts here! You may consider abstracting some of this functionality out to make future calls more dry, but in the case of this mid mod, there really wasn't anything to dry up. For ruby convention, as well as more readability, I'd suggest looking into using syntax when making your Faraday request. For example: 

```ruby
    Faraday.new(url: 'https://api.nal.usda.gov') do |faraday|
      faraday.params['api_key'] = ENV['FOOD_API_KEY']
      faraday.params['query'] = search_terms
    end

```